[2025-05-23T09:19:44.827+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-23T09:19:44.838+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: x_videos_scraper_dag.get_links_task manual__2025-05-23T09:19:44.110821+00:00 [queued]>
[2025-05-23T09:19:44.843+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: x_videos_scraper_dag.get_links_task manual__2025-05-23T09:19:44.110821+00:00 [queued]>
[2025-05-23T09:19:44.843+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 1
[2025-05-23T09:19:44.851+0000] {taskinstance.py:2879} INFO - Executing <Task(PythonOperator): get_links_task> on 2025-05-23 09:19:44.110821+00:00
[2025-05-23T09:19:44.855+0000] {standard_task_runner.py:72} INFO - Started process 272169 to run task
[2025-05-23T09:19:44.858+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'x_videos_scraper_dag', 'get_links_task', 'manual__2025-05-23T09:19:44.110821+00:00', '--job-id', '6652', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_x.py', '--cfg-path', '/tmp/tmpp938ih0l']
[2025-05-23T09:19:44.859+0000] {standard_task_runner.py:105} INFO - Job 6652: Subtask get_links_task
[2025-05-23T09:19:44.895+0000] {task_command.py:467} INFO - Running <TaskInstance: x_videos_scraper_dag.get_links_task manual__2025-05-23T09:19:44.110821+00:00 [running]> on host f3ad03241d5f
[2025-05-23T09:19:44.951+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='x_videos_scraper_dag' AIRFLOW_CTX_TASK_ID='get_links_task' AIRFLOW_CTX_EXECUTION_DATE='2025-05-23T09:19:44.110821+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-23T09:19:44.110821+00:00'
[2025-05-23T09:19:44.952+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-05-23T09:20:00.686+0000] {taskinstance.py:3301} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/opt/airflow/dags/pipeline_x.py", line 30, in run_x_videos_scraper
    return x_videos_scraper(
  File "/opt/airflow/dags/tasks/x_videos_scraper.py", line 24, in x_videos_scraper
    page.goto(f"https://x.com/{id}/media", timeout=15000)
  File "/home/airflow/.local/lib/python3.10/site-packages/playwright/sync_api/_generated.py", line 9024, in goto
    self._sync(
  File "/home/airflow/.local/lib/python3.10/site-packages/playwright/_impl/_sync_base.py", line 115, in _sync
    return task.result()
  File "/home/airflow/.local/lib/python3.10/site-packages/playwright/_impl/_page.py", line 552, in goto
    return await self._main_frame.goto(**locals_to_params(locals()))
  File "/home/airflow/.local/lib/python3.10/site-packages/playwright/_impl/_frame.py", line 145, in goto
    await self._channel.send("goto", locals_to_params(locals()))
  File "/home/airflow/.local/lib/python3.10/site-packages/playwright/_impl/_connection.py", line 61, in send
    return await self._connection.wrap_api_call(
  File "/home/airflow/.local/lib/python3.10/site-packages/playwright/_impl/_connection.py", line 528, in wrap_api_call
    raise rewrite_error(error, f"{parsed_st['apiName']}: {error}") from None
playwright._impl._errors.TimeoutError: Page.goto: Timeout 15000ms exceeded.
Call log:
  - navigating to "https://x.com/ManUtd/media", waiting until "load"

[2025-05-23T09:20:00.694+0000] {taskinstance.py:1225} INFO - Marking task as FAILED. dag_id=x_videos_scraper_dag, task_id=get_links_task, run_id=manual__2025-05-23T09:19:44.110821+00:00, execution_date=20250523T091944, start_date=20250523T091944, end_date=20250523T092000
[2025-05-23T09:20:00.704+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-05-23T09:20:00.705+0000] {standard_task_runner.py:124} ERROR - Failed to execute job 6652 for task get_links_task (Page.goto: Timeout 15000ms exceeded.
Call log:
  - navigating to "https://x.com/ManUtd/media", waiting until "load"
; 272169)
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/utils/cli.py", line 115, in wrapper
    return f(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 2995, in _run_raw_task
    return _run_raw_task(
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 3149, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 3173, in _execute_task
    return _execute_task(self, context, task_orig)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/opt/airflow/dags/pipeline_x.py", line 30, in run_x_videos_scraper
    return x_videos_scraper(
  File "/opt/airflow/dags/tasks/x_videos_scraper.py", line 24, in x_videos_scraper
    page.goto(f"https://x.com/{id}/media", timeout=15000)
  File "/home/airflow/.local/lib/python3.10/site-packages/playwright/sync_api/_generated.py", line 9024, in goto
    self._sync(
  File "/home/airflow/.local/lib/python3.10/site-packages/playwright/_impl/_sync_base.py", line 115, in _sync
    return task.result()
  File "/home/airflow/.local/lib/python3.10/site-packages/playwright/_impl/_page.py", line 552, in goto
    return await self._main_frame.goto(**locals_to_params(locals()))
  File "/home/airflow/.local/lib/python3.10/site-packages/playwright/_impl/_frame.py", line 145, in goto
    await self._channel.send("goto", locals_to_params(locals()))
  File "/home/airflow/.local/lib/python3.10/site-packages/playwright/_impl/_connection.py", line 61, in send
    return await self._connection.wrap_api_call(
  File "/home/airflow/.local/lib/python3.10/site-packages/playwright/_impl/_connection.py", line 528, in wrap_api_call
    raise rewrite_error(error, f"{parsed_st['apiName']}: {error}") from None
playwright._impl._errors.TimeoutError: Page.goto: Timeout 15000ms exceeded.
Call log:
  - navigating to "https://x.com/ManUtd/media", waiting until "load"

[2025-05-23T09:20:00.715+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 1
[2025-05-23T09:20:00.730+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-23T09:20:00.733+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
